beginner_source/quickstart/build_model_tutorial.py

```diff
@@ -0,0 +1,193 @@
+"""
+Build the Neural Network
+===================
+"""
+
+#################################################################
+# Get Started Building the Model
+# -----------------
+#
+# The data has been loaded and transformed we can now build the model. 
+# We will leverage `torch.nn <https://pytorch.org/docs/stable/nn.html>`_ predefined layers that PyTorch has that can simplify our code.
+# 
+# In the below example, for our FashionMNIT image dataset, we are using a `Sequential` 
+# container from class `torch.nn. Sequential <https://pytorch.org/docs/stable/generated/torch.nn.Sequential.html>`_ 
+# that allows us to define the model layers inline. In the "Sequential" in-line model building format the ``forward()`` 
+# method is created for you and the modules you add are passed in as a list or dictionary in the order that are they are defined.
+# 
+# Another way to bulid this model is with a class 
```

Comment:
Let's start with the class-oriented module because this is the standard approach. You can mention the inline approach as a quick (hacky?) way to put together a network.
```
# In this example,, we build the model class using `nn.Module` ... 

## SHOW nn.Module CODE

# Another way to build this model is using a Sequential container....

## SHOW EQUIVALENT INLINE CODe

#  big plus with using a class that inherits ``nn.Module`` ...
```